Gems/AWSCore/Code/Tools/ResourceMappingTool/README.md

```diff
@@ -89,3 +89,13 @@ you can create the virtualenv manually.
    ```
    $ python3 resource_mapping_tool.py
    ```
+## Tool Arguments
+* `--binaries_path` **[Optional]** Path to QT Binaries necessary for PySide,
```

Comment:
Not for this CR but its more common to have parameter names with 'dashes' than underscores in python. ie "binaries-path", "config-path". This is the pattern followed by most of our script tools like o3de.py